packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -569,10 +569,14 @@ class Message {
 // Represents the contents of one ARB file.
 class AppResourceBundle {
   factory AppResourceBundle(File file) {
-    // Assuming that the caller has verified that the file exists and is readable.
```

Comment:
I think we should keep this comment, just put it above the factory constructor declaration.